frontend/src/views/AppView.vue

```diff
@@ -213,9 +213,16 @@ const config = ref(searchConfigStore.searchConfig);
 watch(
   config, // ref to searchConfigStore.searchConfig
   () => {
-    state.isLoading = true;
-    refreshDataTable();
-    state.isLoading = false;
+    // We set the dataRow to empty here for a better animation.
+    // Otherwise the loading circle would appear right in the middle of the data table, which may be elusive.
+    state.dataRow = [];
+    nextTick(() => {
```

Comment:
we don't need this nextTick probably

Reply:
I did some experiment and found out that the animation is better when using time out. 
So I change nextTick to setTimeout with 100ms.